lighthouse-core/report/v2/report-renderer.js

```diff
@@ -213,18 +256,22 @@ window.ReportRenderer = class ReportRenderer {
    */
   _renderAudit(audit) {
     const element = this._createElement('div', 'lighthouse-audit');
-    let title = audit.result.description;
-    if (audit.result.displayValue) {
-      title += ': ' + audit.result.displayValue;
-    }
-    element.appendChild(this._renderScore(audit.score, title, audit.result.helpText));
+    element.appendChild(this._renderAuditScore(audit));
+
+    // Append audit details to to header section so the entire audit is within one <details>.
+    const header = element.querySelector('.lighthouse-score__header');
```

Comment:
since this depends on markup introduced by `_renderAuditScore`, move header/details stuff in there?